src/network/core/tcp_content.cpp

```diff
@@ -58,7 +58,7 @@ bool ContentInfo::IsValid() const
  */
 std::optional<std::string> ContentInfo::GetTextfile(TextfileType type) const
 {
-	if (this->state == INVALID) return std::nullopt;
+	if (this->state == State::Invalid) return std::nullopt;
```

Comment:
Unlike the changes above, this doesn't explicitly state its `ContentInfo::` namespace.